Failed to build coreclr with ARM softfp

When I tried to build coreclr with ARM softfp, it is failed.
This is happening because TOOLCHAIN is not defined in the case of ARM softf.

error log:
```
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- The ASM compiler identification is Clang
-- Found assembler: /usr/bin/clang-3.5
CMake Error at CMakeLists.txt:87 (message):
  objcopy not found
```

CMakeLists.txt
```
 74       # Ensure that objcopy is present                                          
 75       if(DEFINED ENV{CROSSCOMPILE})                                             
 76         if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)                              
 77           find_program(OBJCOPY ${TOOLCHAIN}-objcopy)                            
 78         elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64)                         
 79           find_program(OBJCOPY ${TOOLCHAIN}-objcopy)                            
 80         else()                                                                  
 81           clr_unknown_arch()                                                    
 82         endif()                                                                 
 83       else()                                                                    
 84         find_program(OBJCOPY objcopy)                                           
 85       endif()                                                                   
 86       if (OBJCOPY STREQUAL "OBJCOPY-NOTFOUND")                                  
 87           message(FATAL_ERROR "objcopy not found")                              
 88       endif()                                                                   
```

cross/arm/toolchain.cmake
```
  7 ## Specify the toolchain                                                        
  8 set(TOOLCHAIN "arm-linux-gnueabihf")                                            
  9 set(TOOLCHAIN_PREFIX ${TOOLCHAIN}-)    
```

cross/arm-softfp/toolchain.cmake
No specified.

--- a/cross/arm-softfp/toolchain.cmake
+++ b/cross/arm-softfp/toolchain.cmake
@@ -4,14 +4,24 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR armv7l)
 
+## Specify the toolchain
+set(TOOLCHAIN "arm-linux-gnueabi")
+set(TOOLCHAIN_PREFIX ${TOOLCHAIN}-)
+#set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}gcc)
+#set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}g++)
+#set(CMAKE_ASM_COMPILER ${TOOLCHAIN_PREFIX}as)
+#set(CMAKE_AR ${TOOLCHAIN_PREFIX}ar)
+#set(CMAKE_OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
+#set(CMAKE_OBJDUMP ${TOOLCHAIN_PREFIX}objdump)
+
 add_compile_options(-target armv7-linux-gnueabi)
 add_compile_options(-mthumb)
 add_compile_options(-mfpu=vfpv3)
 add_compile_options(--sysroot=${CROSS_ROOTFS})
 
-set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target arm-linux-gnueabi")
-set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -B${CROSS_ROOTFS}/usr/lib/arm-linux-gnueabi")
-set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -L${CROSS_ROOTFS}/lib/arm-linux-gnueabi -L${CROSS_ROOTFS}/usr/lib/arm-linux-gnueabi")
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -B${CROSS_ROOTFS}/usr/lib/${TOOLCHAIN}")
+set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -L${CROSS_ROOTFS}/lib/${TOOLCHAIN} -L${CROSS_ROOTFS}/usr/lib/${TOOLCHAIN}")
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --sysroot=${CROSS_ROOTFS}")
 
 set(CMAKE_EXE_LINKER_FLAGS    "${CMAKE_EXE_LINKER_FLAGS}    ${CROSS_LINK_FLAGS}" CACHE STRING "" FORCE)

